# header
Header header 

# jointangle
float32 phi1_lf
float32 phi2_lf
float32 phi1_rf
float32 phi2_rf
float32 phi1_rr
float32 phi2_rr
float32 phi1_lr
float32 phi2_lr